recipes/PyCm/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "pycm" %}
+{% set version = "v7r5p1" %}
+
+# handle Virgo version number format
+{% set semver_version = version|
+    replace('v', '', 1)|
+    replace('r', '.', 1)|
+    replace('p', '.', 1)
+ %}
```

Actually, sorry @rhyspoulton, but it strikes me that you're not using the upstream Virgo `tag` at all, so you can just leave this as 

```suggestion
{% set version = "7.5.1" %}
```

---

No worries, yes it makes the most sense just to remove it.